lte/gateway/python/magma/mobilityd/ip_descriptor_map.py

```diff
@@ -48,7 +48,8 @@ class IpDescriptorMap:
 
     def __init__(self,
                  persist_to_redis: bool = True,
-                 redis_port: int = 6379):
+                 redis_port: int = 6379,
```

Comment:
nit: not related to this diff, can you update the default port to 6380? We are using that for all of the service, I think we missed updating mobilityd to use it too.